lib/ansible/modules/packaging/os/yum.py

```diff
@@ -31,11 +31,16 @@
 version_added: historical
 short_description: Manages packages with the I(yum) package manager
 description:
-     - Installs, upgrade, removes, and lists packages and groups with the I(yum) package manager.
+     - Installs, upgrade, downgrades, removes, and lists packages and groups with the I(yum) package manager.
 options:
   name:
     description:
-      - "Package name, or package specifier with version, like C(name-1.0). When using state=latest, this can be '*' which means run: yum -y update. You can also pass a url or a local path to a rpm file (using state=present).  To operate on several packages this can accept a comma separated list of packages or (as of 2.0) a list of packages."
+      - Package name, or package specifier with version, like C(name-1.0).
+        A previous version may be specified C(name-0.9) to downgrade a package.
+        When using state=latest, this can be '*' which means run C(yum -y update).
+        You can also pass a url or a local path to a rpm file (using state=present).
+        To operate on several packages this can accept a comma separated list
+        of packages or (as of 2.0) a list of packages.
```

Comment:
In light of needing to set allow_downgrade, maybe something like

```
      - Package name, or package specifier with version, like C(name-1.0).
	If a previous version is specified, the task also needs to turn
	C(allow_downgrade) on. See the C(allow_downgrade) documentation for
	caveats with downgrading packages.  A previous version may be
	specified C(name-0.9) to downgrade a package.  When using
	state=latest, this can be '*' which means run C(yum -y update).  You
	can also pass a url or a local path to a rpm file (using
	state=present).  To operate on several packages this can accept
	a comma separated list of packages or (as of 2.0) a list of packages.
```